src/Sulu/Bundle/MediaBundle/Resources/config/lists/media.xml

```diff
@@ -8,6 +8,23 @@
             <field-name>%sulu.model.media.class%.type</field-name>
         </join>
     </joins>
+    <joins name="previewImageFileVersion">
+        <join>
+            <entity-name>PreviewImage_%sulu.model.media.class%</entity-name>
```

Comment:
would name this just `previewImage`

Reply:
not really euphoric about any of the two, so i dont care about that 😉 will rename it if @danrot is okay with `previewImage ` too

Reply:
Hm, until now we always used names of classes here, but we never had to link the same entitiy itself as far as i can remember... I woul dhav ea tendency to keep the class name in there and probably would have suffixed the `PreviewImage` part. This way it would still be clear what exactly is loaded when looking at the queries. Might be obvious in this case anyway, but if we want to use the same conventions in other places it might not.

Also, until now all entity names have the class included in some way.

Reply:
Oh, and I was wrong, found a similar use case: https://github.com/sulu/sulu/blob/0a08318/src/Sulu/Bundle/CategoryBundle/Resources/config/lists/categories.xml#L21

Reply:
adjusted the `entity-name` to use this format: `%sulu.model.media.class%PreviewImage`